Gemfile

```diff
@@ -1,13 +1,13 @@
 source 'https://rubygems.org'
 git_source(:github) { |repo_name| "https://github.com/#{repo_name}.git" }
 
-ruby '~> 2.6.5'
+ruby '~> 2.6'
```

Comment:
Was this intended?

Previously: #5020, #5018

Reply:
gets me every time, thank you 😓

Reply:
https://github.com/18F/identity-idp/pull/5063